ehr-hard-deletion-lambda/EhrHardDeletion.py

```diff
@@ -36,8 +36,11 @@ def delete_ehr_from_s3(inbound_conversation_id: str) -> None:
         repo_bucket = s3.Bucket(s3_bucket_name)
 
         if list(repo_bucket.objects.filter(Prefix=inbound_conversation_id + "/")):
-            logger.info("Attempting to delete EHR in the S3 Bucket")
-            repo_bucket.objects.filter(Prefix=inbound_conversation_id + "/").delete()
+            count_of_files = 0
+            for unused_var in repo_bucket.objects.filter(Prefix=inbound_conversation_id + "/"): count_of_files+=1
+            logger.info(f"Attempting to delete EHR in the S3 Bucket ({str(count_of_files)} file(s))")
```

Comment:
probably we could replace line 38-40 like this and reduce one .filter() call
```python
        count_of_files = len(list(repo_bucket.objects.filter(Prefix=inbound_conversation_id + "/")))
        if count_of_files > 0:
            logger.info(f"Attempting to delete EHR in the S3 Bucket ({str(count_of_files)} file(s))")
            ...

Reply:
I tried `len()` but got an error... so changed it to what it is now... but I didn't try `len(list(`...